bin/readme.txt

```diff
@@ -0,0 +1,43 @@
+=== WordPress REST API (Version 2) ===
+Contributors: rmccue, rachelbaker
+Tags: json, rest, api, rest-api
+Requires at least: 4.3-alpha
+Tested up to: 4.3-beta
+Stable tag: {{TAG}}
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+Access your site's data through an easy-to-use HTTP REST API. (Version 2)
+
+== Description ==
+WordPress is moving towards becoming a fully-fledged application framework, and we need new APIs. This project was born to create an easy-to-use, easy-to-understand and well-tested framework for creating these APIs, plus creating APIs for core.
+
+This plugin provides an easy to use REST API, available via HTTP. Grab your site's data in simple JSON format, including users, posts, taxonomies and more. Retrieving or updating data is as simple as sending a HTTP request.
+
+Want to get your site's posts? Simply send a `GET` request to `/wp-json/wp/v2/posts`. Update user with ID 4? Send a `PUT` request to `/wp-json/wp/v2/users/4`. Get all posts with the search term "awesome"? `GET /wp-json/wp/v2/posts?filter[s]=awesome`. It's that easy.
```

Comment:
I know. :) IIRC, we were planning on moving WP_Query args back under `filter` and adding our own set to the top level?